### PR TITLE
[zh-cn] Resync content/zh-cn/docs/reference/networking/virtual-ips.md

### PR DESCRIPTION
**Pull request by mengjiao-liu**
### Issue

Closes: #48245

See [the preview page](https://deploy-preview-48249--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/networking/virtual-ips/).

### CODE REVIEW

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 1]
```diff
@@ -497,19 +475,149 @@ the node before starting kube-proxy.
...
+
+<!--
+The nftables API is the successor to the iptables API and is designed
+to provide better performance and scalability than iptables. The
+`nftables` proxy mode is able to process changes to service endpoints
+faster and more efficiently than the `iptables` mode, and is also able
+to more efficiently process packets in the kernel (though this only
+becomes noticeable in clusters with tens of thousands of services).
+-->
+nftables API 是 iptables API 的继任者，旨在提供比 iptables 更好的性能和可扩展性。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:523]:
```suggestion
nftables API 是 iptables API 的后继，旨在提供比 iptables 更好的性能和可扩展性。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 2]
```diff
@@ -497,19 +475,149 @@ the node before starting kube-proxy.
...
+#### Migrating from `iptables` mode to `nftables`
+
+Users who want to switch from the default `iptables` mode to the
+`nftables` mode should be aware that some features work slightly
+differently the `nftables` mode:
+-->
+#### 从 `iptables` 模式到 `nftables` 模式的迁移 {#migrating-from-iptables-mode-to-nftables}
+
+想要从默认的 `iptables` 模式切换到 `nftables` 模式的用户应注意，在
+`nftables` 模式下，一些功能的工作方式略有不同：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:546]:
```suggestion
`nftables` 模式下，一些特性的工作方式略有不同：
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 3]
```diff
@@ -497,19 +475,149 @@ the node before starting kube-proxy.
...
+   blocked by a firewall. This approach will not work with firewalls
+   based on nftables, so kube-proxy's `nftables` mode does not do
+   anything here; if you have a local firewall, you must ensure that
+   it is properly configured to allow Kubernetes traffic through
+   (e.g., by allowing inbound traffic on the entire NodePort range).
+-->
+- **NodePort 与防火墙的交互**：kube-proxy 的 `iptables` 模式尝试与过于激进的防火墙兼容；
+  对于每个 NodePort 服务，它会添加规则以接受该端口的入站流量，以防该流量被防火墙阻止。
+  这种方法不适用于基于 nftables 的防火墙，因此 kube-proxy 的 `nftables` 模式在这里不会做任何事情；
+  如果你有本地防火墙，必须确保其正确配置以允许 Kubernetes 流量通过（例如，允许整个 NodePort 范围的入站流量）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:598]:
```suggestion
  如果你有本地防火墙，必须确保其配置正确以允许 Kubernetes 流量通过（例如，允许整个 NodePort 范围的入站流量）。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 4]
```diff
@@ -497,19 +475,149 @@ the node before starting kube-proxy.
...
+   `iptables` mode of kube-proxy installs a workaround for this bug,
+   but this workaround was later found to cause other problems in some
+   clusters. The `nftables` mode does not install any workaround by
+   default, but you can check kube-proxy's
+   `iptables_ct_state_invalid_dropped_packets_total` metric to see if
+   your cluster is depending on the workaround, and if so, you can run
+   kube-proxy with the option `--conntrack-tcp-be-liberal` to work
+   around the problem in `nftables` mode.
+-->
+- **Conntrack 错误修复**：6.1 之前的 Linux 内核存在一个错误，可能导致与服务 IP 的长时间
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:613]:
```suggestion
- **Conntrack 错误的规避**：6.1 之前的 Linux 内核存在一个错误，可能导致与服务 IP 的长时间
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 5]
```diff
@@ -497,19 +475,149 @@ the node before starting kube-proxy.
...
+  TCP 连接被关闭，并出现“对等方重置连接”的错误。kube-proxy 的 `iptables`
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:614]:
```suggestion
  TCP 连接被关闭，并出现“Connection reset by peer（对方重置连接）”的错误。kube-proxy 的 `iptables`
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 6]
```diff
@@ -497,19 +475,149 @@ the node before starting kube-proxy.
...
+  模式为此错误安装了一个修复程序，但后来发现该修复程序在某些集群中会导致其他问题。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:615]:
```suggestion
  模式为此错误配备了一个修复程序，但后来发现该修复程序在某些集群中会导致其他问题。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 7]
```diff
@@ -684,50 +792,67 @@ fail with a message indicating an IP address could not be allocated.
...
+the control plane replaces the existing etcd allocator with a revised implementation
+that uses IPAddress and ServiceCIDR objects instead of an internal global allocation map.
+Each cluster IP address associated to a Service then references an IPAddress object.
 -->
 如果你启用 `MultiCIDRServiceAllocator` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gate/)和
 [`networking.k8s.io/v1alpha1` API 组](/zh-cn/docs/tasks/administer-cluster/enable-disable-api/)，
-控制平面将用一个新的分配器替换现有的 etcd 分配器，使用 IPAddress 对象而不是内部的全局分配映射。
-与每个 Service 关联的 ClusterIP 地址将有一个对应的 IPAddress 对象。
+控制平面用一个改进后的实现替换现有的 etcd 分配器，使用 IPAddress 和 ServiceCIDR
+对象而不是内部的全局分配映射。与每个 Service 关联的 ClusterIP 地址将有一个对应的
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:819]:
```suggestion
对象而不是内部的全局分配映射。与某 Service 关联的每个 ClusterIP 地址将有一个对应的
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 8]
```diff
@@ -684,50 +792,67 @@ fail with a message indicating an IP address could not be allocated.
...
-users can use masks equal or larger than /64 (previously it was /108).
+Making IP address allocations available via the API means that you as a cluster administrator
+can allow users to inspect the IP addresses assigned to their Services.
+Kubernetes extensions, such as the [Gateway API](/docs/concepts/services-networking/gateway/),
+can use the IPAddress API to extend Kubernetes' inherent networking capabilities.
 -->
-新分配器的主要好处之一是它取消了对 `service-cluster-ip-range` 的大小限制，对 IPv4 没有大小限制，
-对于 IPv6 用户可以使用等于或大于 /64 的掩码（以前是 /108）。
+通过 API 提供 IP 地址分配，意味着作为集群管理员，你可以允许用户检查分配给他们的 Service 的 IP 地址。
+Kubernetes 扩展，（例如 [Gateway API](/zh-cn/docs/concepts/services-networking/gateway/)）
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:849]:
```suggestion
Kubernetes 扩展（例如 [Gateway API](/zh-cn/docs/concepts/services-networking/gateway/)）
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 9]
```diff
@@ -747,11 +872,56 @@ NAME              PARENTREF
 2001:db8:1:2::1   services/default/kubernetes
 2001:db8:1:2::a   services/kube-system/kube-dns
 ```
+<!--
+Kubernetes also allow users to dynamically define the available IP ranges for Services using
+ServiceCIDR objects. During bootstrap, a default ServiceCIDR object named `kubernetes` is created
+from the value of the `--service-cluster-ip-range` command line argument to kube-apiserver:
+-->
+Kubernetes 还允许用户使用 ServiceCIDR 对象动态定义 Service 的可用 IP 范围。在引导过程中，会根据
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:880]:
```suggestion
Kubernetes 还允许用户使用 ServiceCIDR 对象动态定义 Service 的可用 IP 范围。在引导过程中，集群会根据
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 10]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+
+<!--
+1. kube-proxy is healthy, meaning:
+   - it's able to progress programming the network and isn't timing out while doing
+     so (the timeout is defined to be: **2 × `iptables.syncPeriod`**); and
+2. the node is not being deleted (there is no deletion timestamp set for the Node).
+-->
+1. kube-proxy 是健康的，意味着：
+    - 它能够继续进行网络编程，并且在此过程中不会超时（超时时间定义为：**2 × `iptables.syncPeriod`**）；并且
+2. 节点没有被删除（节点上没有设置删除时间戳）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1021]:
```suggestion
2. 节点没有被删除（Node 对象上没有设置删除时间戳）。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 11]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+
+<!--
+The reason why kube-proxy returns 503 and marks the node as not
+eligible when it's being deleted, is because kube-proxy supports connection
+draining for terminating nodes. A couple of important things occur from the point
+of view of a Kubernetes-managed load balancer when a node _is being_ / _is_ deleted.
+-->
+kube-proxy 在节点被删除时返回 503 并将节点标记为不符合条件的原因在于
+kube-proxy 对终止的节点支持连接腾空。从 Kubernetes 管理的负载均衡器的角度来看，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1030]:
```suggestion
kube-proxy 对处于终止过程中的节点支持连接腾空。从 Kubernetes 管理的负载均衡器的角度来看，
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 12]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+<!--
+While deleting:
+
+* kube-proxy will start failing its readiness probe and essentially mark the
+   node as not eligible for load balancer traffic. The load balancer health
+   check failing causes load balancers which support connection draining to
+   allow existing connections to terminate, and block new connections from
+   establishing.
+-->
+当节点删除时：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1042]:
```suggestion
当节点被删除时：
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 13]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+
+* kube-proxy 将开始失败其就绪探针，并将该节点标记为不符合负载均衡器流量的条件。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1044]:
```suggestion
* kube-proxy 的就绪探针将开始失败，并将该节点标记为不胜任接收负载均衡器流量。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 14]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+<!--
+When deleted:
+
+* The service controller in the Kubernetes cloud controller manager removes the
+  node from the referenced set of eligible targets. Removing any instance from
+  the load balancer's set of backend targets immediately terminates all
+  connections. This is also the reason kube-proxy first fails the health check
+  while the node is deleting.
+-->
+当节点删除后：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1057]:
```suggestion
当节点被删除后：
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 15]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+
+* Kubernetes 云控制器管理器中的服务控制器会将节点从引用的合格目标集中移除。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1059]:
```suggestion
* Kubernetes 云控制器管理器中的服务控制器会将节点从所引用的候选目标集中移除。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 16]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+`/livez` 路径**不**考虑节点的删除状态，仅考虑其网络编程进度。因此，对于任何希望为
+ kube-proxy 定义存活探针的人来说，推荐使用 `/livez` 路径。
+
+<!--
+Users deploying kube-proxy can inspect both the readiness / liveness state by
+evaluating the metrics: `proxy_livez_total` / `proxy_healthz_total`. Both
+metrics publish two series, one with the 200 label and one with the 503 one.
+-->
+部署 kube-proxy 的用户可以通过评估指标 `proxy_livez_total` / `proxy_healthz_total`
+来检查就绪/存活状态。这两个指标都发布了两个系列，一个带有 200 标签，另一个带有 503 标签。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1084]:
```suggestion
来检查就绪/存活状态。这两个指标都发布了两个序列，一个带有 200 标签，另一个带有 503 标签。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 17]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+
+1. kube-proxy is healthy/ready, and
+2. has a local endpoint on the node in question.
+
+Node deletion does **not** have an impact on kube-proxy's return
+code for what concerns load balancer health checks. The reason for this is:
+deleting nodes could end up causing an ingress outage should all endpoints
+simultaneously be running on said nodes.
+-->
+对于 `Local` 服务：如果满足以下条件，kube-proxy 将返回 200：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1097]:
```suggestion
对于 `Local` Service：如果满足以下条件，kube-proxy 将返回 200：
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 18]
```diff
@@ -837,6 +997,122 @@ relevant Service.
...
+
+1. kube-proxy 是健康/就绪的，并且
+2. 在相关节点上有一个本地端点。
+
+对于负载均衡器健康检查而言，节点删除**不会**对 kube-proxy
+的返回代码产生影响。原因是：如果所有端点同时在上述节点上运行，则删除节点最终可能会导致入口中断。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1103]:
```suggestion
的返回代码产生影响。原因是：如果所有端点同时在上述节点上运行，则删除节点最终可能会导致入站流量中断。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 19]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+: For kube-proxy, this means prioritizing sending traffic to endpoints within
+  the same zone as the client. The EndpointSlice controller updates
+  EndpointSlices with `hints` to communicate this preference, which kube-proxy
+  then uses for routing decisions. If a client's zone does not have any
+  available endpoints, traffic will be routed cluster-wide for that client.
+
+In the absence of any value for `trafficDistribution`, the default routing
+strategy for kube-proxy is to distribute traffic to any endpoint in the cluster.
+-->
+`PreferClose` 与 kube-proxy
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1193]:
```suggestion
`PreferClose` 与 kube-proxy 结合
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 20]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+: 对于 kube-proxy，这意味着优先将流量发送到与客户端位于同一区域的端点。
+  EndpointSlice 控制器更新带有 `hints` 的 EndpointSlices 以传达此偏好，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1195]:
```suggestion
  EndpointSlice 控制器使用 `hints` 来更新 EndpointSlices 以传达此偏好，
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 21]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+  kube-proxy 然后使用这些提示进行路由决策。如果客户端的区域没有任何可用的端点，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1196]:
```suggestion
  之后，kube-proxy 会使用这些提示进行路由决策。如果客户端的区域没有可用的端点，
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 22]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+  则流量将在整个集群范围内路由。
+
+如果 `trafficDistribution` 没有任何值，kube-proxy 的默认路由策略是将流量分配到集群中的任何端点。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1199]:
```suggestion
如果 `trafficDistribution` 没有任何值，kube-proxy 的默认路由策略是将流量分配到集群中的任一端点。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 23]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+  includes safeguards (such as the [fallback
+  behavior](/docs/concepts/services-networking/topology-aware-routing/#three-or-more-endpoints-per-zone)
+  for small numbers of endpoints) and could lead to the feature being disabled
+  in certain scenarios for load-balancing reasons. This approach sacrifices some
+  predictability in favor of potential load balancing.
+-->
+* `service.kubernetes.io/topology-mode: Auto`：尝试根据可分配的 CPU
+  资源在各区域之间按比例分配流量。此启发式方法包括一些保障措施
+  （例如针对少量端点的[回退行为](/zh-cn/docs/concepts/services-networking/topology-aware-routing/#three-or-more-endpoints-per-zone)），
+  并可能在某些负载均衡原因的场景下导致该特性被禁用。这种方法在一定程度上牺牲了可预测性，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1226]:
```suggestion
  并在某些场景下可能因负载均衡原因导致该特性被禁用。这种方法在一定程度上牺牲了可预测性，
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 24]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+
+<!--
+* `trafficDistribution: PreferClose`: This approach aims to be slightly simpler
+  and more predictable: "If there are endpoints in the zone, they will receive
+  all traffic for that zone, if there are no endpoints in a zone, the traffic
+  will be distributed to other zones". While the approach may offer more
+  predictability, it does mean that you are in control of managing a [potential
+  overload](#considerations-for-using-traffic-distribution-control).
+-->
+* `trafficDistribution: PreferClose`：这种方法旨在更简单和更可预测：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1237]:
```suggestion
* `trafficDistribution: PreferClose`：这种方法偏重更简单和更可预测：
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 25]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+
+When compared to the `trafficDistribution` field, the traffic policy fields
+(`externalTrafficPolicy` and `internalTrafficPolicy`) are meant to offer a
+stricter traffic locality requirements. Here's how `trafficDistribution`
+interacts with them:
+-->
+### 与流量策略的交互 {#interaction-with-traffic-policies}
+
+与 `trafficDistribution` 字段相比，流量策略字段
+（`externalTrafficPolicy` 和 `internalTrafficPolicy`）旨在提供更严格的流量本地化要求。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1260]:
```suggestion
（`externalTrafficPolicy` 和 `internalTrafficPolicy`）旨在提供更严格的流量局域化要求。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 26]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+以下是 `trafficDistribution` 与它们的交互方式：
+
+<!--
+* Precedence of Traffic Policies: For a given Service, if a traffic policy
+  (`externalTrafficPolicy` or `internalTrafficPolicy`) is set to `Local`, it
+  takes precedence over `trafficDistribution: PreferClose` for the corresponding
+  traffic type (external or internal, respectively).
+-->
+* 流量策略的优先级：对于给定的 Service，如果流量策略
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1269]:
```suggestion
* 流量策略的优先序：对于给定的 Service，如果流量策略
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 27]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+* **端点过载的概率增加：** `PreferClose` 启发式方法将尝试将流量路由到最近的健康端点，
+  而不是将流量均匀分布到所有端点。如果某个区域内的端点数量不足，它们可能会过载。
+  如果传入流量在各区域之间分布不均，这种情况更有可能发生。为减轻这种情况，请考虑以下策略：
+
+    * [Pod 拓扑分布约束](/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints/):
+      使用 Pod 拓扑分布约束在各区域之间更均匀地分布你的 Pod。
+
+    * 区域特定的 Deployment：如果你预计会看到不均衡的流量模式，
+      为每个区域创建一个单独的 Deployment。这种方法允许独立扩展各个工作负载。
+      生态系统中还有一些 Kubernetes 项目之外的工作负载管理插件，可以在这方面提供帮助。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1319]:
```suggestion
  * [Pod 拓扑分布约束](/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints/)：
    使用 Pod 拓扑分布约束在各区域之间更均匀地分布你的 Pod。

  * 区域特定的 Deployment：如果你预计会看到不均衡的流量模式，
    可以为每个区域创建一个单独的 Deployment。这种方法允许独立扩展各个工作负载。
    生态系统中还有一些 Kubernetes 项目之外的工作负载管理插件，可以在这方面提供帮助。
```

File: content/zh-cn/docs/reference/networking/virtual-ips.md [hunk 28]
```diff
@@ -884,6 +1160,173 @@ pool.
...
+
+<!--
+* **Implementation-specific behavior:** Each dataplane implementation may handle
+  this field slightly differently. If you're using an implementation other than
+  kube-proxy, refer the documentation specific to that implementation to
+  understand how this field is being handled.
+-->
+* **实现特定行为：** 每个数据平面实现可能会稍有不同地处理此字段。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/networking/virtual-ips.md:1327]:
```suggestion
* **特定于具体实现的行为：** 各个数据平面实现处理此字段的方式可能会稍有不同。
```

**Comment by mengjiao-liu** [comment]
All comments addressed. Thanks @tengqm for the pieces of advice.

**Pull request merged**